app/controllers/webchat.js

```diff
@@ -20,6 +20,7 @@ export default Ember.Controller.extend({
     zip: "",
     phone: "",
     queue: "",
+    locale: "",
```

Comment:
should this be blank or "en"?  does the chat handle "" properly?

Reply:
That is a good question. I will test but will most likely just default it to "en"

Reply:
After testing, if we do not set it it will default to "en" inside the webchat-js code, but better to just set one. Change coming soon.

Reply:
Don't have much context here, but we've moved to bcp 47 lang tags in most places.  i.e. `en-us`.  Just a heads up, not sure if it will work or matter here.